job-shop-scheduling/demo.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from dwave.system.composites import EmbeddingComposite
+from dwave.system.samplers import DWaveSampler
+
+from job_shop_scheduler import get_jss_bqm
+
+# Construct a BQM for jobs 'a', 'b' and 'c'
+jobs = {"a": [("mixer", 2), ("oven", 1)],
+        "b": [("mixer", 1)],
```

Comment:
Why generic jobs "a", "b", "c" versus real tasks "mixer" and "oven"? Why not have all as real ones?